extras/create_snapshots_reports_scorecard.py

```diff
@@ -365,127 +376,135 @@ def generate_weekly_snapshots(db, cyhy_db_section):
+                    "docker",
+                    "run",
+                    "--rm",
+                    "--volume",
+                    "/etc/cyhy:/etc/cyhy",
+                    "--volume",
+                    "{}:/home/cyhy".format(CYHY_REPORT_DIR),
+                    "{}/cyhy-reports:stable".format(NCATS_DHUB_URL),
+                    "cyhy-report",
+                    "--cyhy-section",
+                    cyhy_db_section,
+                    "--scan-section",
+                    scan_db_section,
+                    "--final",
+                    "--encrypt",
+                    org_id,
+                ],
+                stdout=subprocess.PIPE,
+                stdin=subprocess.PIPE,
+                stderr=subprocess.PIPE,
             )
-            logging.info(
-                "[%s] Stderr report detail: %s%s",
-                threading.current_thread().name,
-                data,
-                err,
+    else:
+        if nolog:
+            p = subprocess.Popen(
+                [
+                    "cyhy-report",
+                    "--nolog",
+                    "--cyhy-section",
+                    cyhy_db_section,
+                    "--scan-section",
+                    scan_db_section,
+                    "--final",
+                    "--encrypt",
+                    org_id,
+                ],
+                stdout=subprocess.PIPE,
+                stdin=subprocess.PIPE,
+                stderr=subprocess.PIPE,
             )
-            failed_reports.append(i)
+        else:
+            p = subprocess.Popen(
+                [
+                    "cyhy-report",
+                    "--cyhy-section",
+                    cyhy_db_section,
+                    "--scan-section",
+                    scan_db_section,
+                    "--final",
+                    "--encrypt",
+                    org_id,
+                ],
+                stdout=subprocess.PIPE,
+                stdin=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+    data, err = p.communicate()
+    report_time = time.time() - report_time
+    report_durations.append((org_id, report_time))
+    return_code = p.returncode
+    if return_code == 0:
+        logging.info(
+            "[%s] Successful report generated: %s (%.2f s)",
+            threading.current_thread().name,
+            org_id,
+            round(report_time, 2),
+        )
+        successful_reports.append(org_id)
+    else:
+        logging.info(
+            "[%s] Failure to generate report: %s",
+            threading.current_thread().name,
+            org_id,
+        )
+        logging.info(
+            "[%s] Stderr report detail: %s%s",
+            threading.current_thread().name,
+            data,
+            err,
+        )
+        failed_reports.append(org_id)
+
+
+def create_all_reports(cyhy_db_section, scan_db_section, use_docker, nolog):
+    """Create reports for all orgs in a list."""
```

Comment:
Same comment about the name/docstring as for `create_all_snapshots()`.

Reply:
Please see https://github.com/cisagov/cyhy-reports/pull/106/commits/c410e90e6457f84bfefe22006790ca0e7a91d910 where I cleaned up the report generation functions along the same lines as what I did for snapshots in https://github.com/cisagov/cyhy-reports/commit/e969421ea40668481a76a06786db823b27d54eb9.